pyramid/config/views.py

```diff
@@ -1222,6 +1259,71 @@ def add_default_view_predicates(self):
             ):
             self.add_view_predicate(name, factory)
 
+    def add_default_accept_view_order(self):
+        for accept in (
+            'text/html',
+            'application/xhtml+xml',
+            'application/xml',
+            'text/xml',
+            'text/plain',
+            'application/json',
+        ):
+            self.add_accept_view_order(accept)
+
+    @action_method
+    def add_accept_view_order(
+        self,
+        value,
+        weighs_more_than=None,
+        weighs_less_than=None,
+    ):
+        """
+        Specify an ordering preference for the ``accept`` view option used
+        during :term:`view lookup`.
+
+        By default, if two views have different ``accept`` options and a
+        request specifies ``Accept: */*`` or omits the header entirely then
+        it is random which view will be selected. This method provides a way
```

Comment:
Another instance of "random", where I'm not sure that's the most accurate term.

Reply:
@stevepiercy the use of the word random here is fine. Ultimately it ends up depending on the order in which your views are added, and the position of the sun and the `PYTHONHASHSEED`.